packages/zk-wizard/src/zk-wallet-account.ts

```diff
@@ -728,6 +734,7 @@ export class ZkWalletAccount {
         .eth()
         .add(fee)
         .toString(),
+      gas: 400000, // TODO : minimum gas set inside 'sendTx' with function signature
```

Comment:
Does the TODO comment mean this logic?
https://github.com/zkopru-network/zkopru/blob/14e0d5147c7e2868b69cb51cf850a95852358615/packages/contracts/src/tx-util.ts#L19-L58

Reply:
lost my comment!

shortly said that I forgot to remove this one. ;)
And 'TODO' means that I want the gas amount is following a 'gasTable' but it is not created yet. 

Removed at c5e2c52